cl_manycore/regression/cuda/test_float_vec_add.c

```diff
@@ -0,0 +1,210 @@
+#include "test_float_vec_add.h"
+
+#define TEST_NAME "test_float_vec_add"
+#define ALLOC_NAME "default_allocator"
+
+/*!
+ * Runs the floating point vector addition a one 2x2 tile groups. A[N] + B[N] --> C[N]
+ * Grid dimensions are prefixed at 1x1. --> block_size_x is set to N.
+ * This tests uses the software/spmd/bsg_cuda_lite_runtime/float_vec_add/ Manycore binary in the BSG Manycore bitbucket repository.  
+*/
+
+
+void host_float_vec_add (float *A, float *B, float *C, int N) { 
+	for (int i = 0; i < N; i ++) { 
+		C[i] = A[i] + B[i];
+	}
+	return;
+}
+
+
+int kernel_float_vec_add () {
+	bsg_pr_test_info("Running the CUDA Floating Point Vector Addition Kernel on a 1x1 grid of 2x2 tile group.\n\n");
```

Comment:
Can you line split down to 80 characters?

Reply:
Here's a good reason why. I shouldn't need to scroll sideways to read your code...

![image](https://user-images.githubusercontent.com/6378612/60825949-af82df00-a161-11e9-96d8-e4822a707b87.png)

Reply:
Very well, I've modified the test file to limit to 80 characters. Let me know if this looks good, and I'll apply the same to all other tests. Thanks Dustin!